src/frontend/komponenter/Behandling/Personopplysninger/InnvandringUtvandring.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { IInnflyttingTilNorge, IUtflyttingFraNorge } from '../../../typer/personopplysninger';
+import { BredTd, KolonneTitler, TabellWrapper } from './TabellWrapper';
+import TabellHeader from './TabellHeader';
+import FlyMedSky from '../../../ikoner/FlyMedSky';
+
+interface Props {
+    innvandringer: IInnflyttingTilNorge[];
+    utvandringer: IUtflyttingFraNorge[];
+}
+
+const InnvandringUtVandring: React.FC<Props> = ({ innvandringer, utvandringer }) => {
+    if (innvandringer.length == 0 && utvandringer.length == 0) {
+        return (
+            <TabellWrapper>
+                <TabellHeader Ikon={FlyMedSky} tittel={'Innvandring og utvandring'} />
+            </TabellWrapper>
+        );
+    }
+
+    return (
+        <TabellWrapper erDobbelTabell>
+            <TabellHeader Ikon={FlyMedSky} tittel={'Innvandring og utvandring'} />
+            {innvandringer.length == 0 && utvandringer.length == 0}
```

Comment:
`{innvandringer.length == 0 && utvandringer.length == 0}` kan slettes eller tas i bruk? 

Hvis man kun har innvandringer nå, så vises headers for utvandringer også, er det ønskelig? Ser att det er "ingen data" på andre steder uten headers

Reply:
Bra observert 😊